README.md

```diff
@@ -10,3 +10,25 @@ Tom Ferdinandt
 Julius Gummersbach  
 Josias Kasemann  
 Taylor Seyler  
+
+## Installation
+
+(this installation was tested on windows using the mingw compiler)
+
+Install Websocketpp:
+1. Clone `https://github.com/zaphoyd/websocketpp` repository in the same directory as the project folder
+
+Install Boost:
+1. Download the zip file from `https://www.boost.org/users/history/version_1_81_0.html`
+2. Unpack it in the same directory as the project folder.
```

Comment:
it needs to be unpacked in a folder caled "boostlibrary" right now. But we should maybe change that instead of this README